msal_extensions/persistence.py

```diff
@@ -188,7 +262,14 @@ def save(self, content):
             self._file_persistence.touch()  # For time_last_modified()
 
     def load(self):
-        return self._agent.load()
+        data = self._agent.load()
+        if data is None:
+            # Lower level libsecret would return None when found nothing. Here
+            # in persistence layer, we convert it to a unified error for consistence.
+            raise PersistenceNotFound(message=(
```

Comment:
My bad. Although all other new PersistenceNotFound are not considered as breaking change, this particular one is. https://github.com/AzureAD/microsoft-authentication-extensions-for-python/pull/67/files#diff-3ef6fda259d01e5bd78b0a594e5f9f01R263-R264

Please abandon the 0.2.3 and let's go with 0.3.0